UPGRADE-6.3.md

```diff
@@ -265,12 +265,6 @@ Sidebar menu items are (in contrast to the classic main menu) created independen
 After migration menu items for backend modules might not yet be migrated to new sidebar menu items. Check the backend
 module list to see if modules are missing. 
 
-All menu items from standard content boxes will be migrated automatically. Custom content boxes need to be migrated
```

Comment:
There is no manual effort needed anymore?

Reply:
As you said today, there already was this migration mechanism (I was confused). For projects, no further action should be required.

I added a note on what to do for third-party bundles.